langstream-runtime/langstream-runtime-impl/src/main/python/langstream/util.py

```diff
@@ -72,8 +72,7 @@ def process(self, records: List[Record]) -> List[Tuple[Record, Union[List[Record
         for record in records:
             try:
                 processed = self.process_record(record)
-                if len(processed) > 0:
-                    results.append((record, processed))
+                results.append((record, processed))
```

Comment:
I think there should always be a result for any input value

Reply:
yes